### ISSUE TITLE
Add report when users switch to empty take lane

### ISSUE THREAD

**Issue by ScottChesworth** [jcsteh/osara#711]
When cycling through takes, we report take number and name. However, when an empty take lane is selected, right now there's no report. Could OSARA speak "Empty lane" or something similar?
Our typical logic is that if nothing changes, there's no report. In this case, the take is changed, so ideally OSARA would report it.

Implementation note from/for @jcsteh:
we can't tell the difference between "no take due to an empty take lane" and "no take for some other reason". I guess we could count the takes, and if there's 1 or more, we know "no take" means "empty take lane".

Thanks in advance.

**Comment by jcsteh** [comment]
Two notes:

1. The message is "empty take lane". That's longer than "empty lane", but I was worried some users might be confused by just "lane" when we're dealing with takes. This shouldn't be a verbosity problem because the potentially superfluous information is after the important part: "empty".
2. I also added a "no takes" message. While nothing happened, this was easy enough to do and I figured it might be useful in the same way that pressing up/down arrow when there are no tracks says "no tracks".
